### PR TITLE
Lint: remove useless val

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**
/kind cleanup

Use unused linter to check, but only for val.

Too many func do not call by code but by hand, this linter detect them as unused, so do not enable linter here.

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/cloud/gettoken.go [hunk 1]
```diff
@@ -17,14 +17,9 @@ import (
 
 var (
 	gettokenLongDescription = `
-"keadm gettoken" command prints the token to use for establishing bidirectional trust between edge nodes and cloudcore. 
-A token can be used when a edge node is about to join the cluster. With this token the cloudcore then approve the 
+"keadm gettoken" command prints the token to use for establishing bidirectional trust between edge nodes and cloudcore.
+A token can be used when a edge node is about to join the cluster. With this token the cloudcore then approve the
 certificate request.
-`
-	gettokenExample = `
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: We shoud use the `gettokenExample` in L39

File: edge/pkg/edged/volume/csi/csi_plugin.go [hunk 2]
```diff
@@ -67,11 +67,9 @@ const (
 	fsTypeBlockName = "block"
 
 	// TODO: increase to something useful
-	csiResyncPeriod = time.Minute
+	//csiResyncPeriod = time.Minute
```
Comment (fisherxu) [inline edge/pkg/edged/volume/csi/csi_plugin.go:70]: delete it here?

File: keadm/cmd/keadm/app/cmd/cloud/gettoken.go (hunk 1 above)
Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: Not sure, see https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/gettoken.go#L31-L51

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/init.go#L61

Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: It is `cloudInitExample` not `gettokenExample `.

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: https://github.com/kubeedge/kubeedge/blob/master/keadm/cmd/keadm/app/cmd/cloud/gettoken.go#L39

Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: Get it.

File: edge/pkg/edged/volume/csi/csi_util.go [hunk 3]
```diff
@@ -35,11 +31,8 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/volume"
 	utilstrings "k8s.io/utils/strings"
-)
-
-const (
-	testInformerSyncPeriod  = 100 * time.Millisecond
-	testInformerSyncTimeout = 30 * time.Second
+	"os"
+	"path/filepath"
```
Comment (fisherxu) [inline edge/pkg/edged/volume/csi/csi_util.go:35]: Move to first category :)

File: keadm/cmd/keadm/app/cmd/cloud/gettoken.go (hunk 1 above)
Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/cloud/gettoken.go:24]: Please don't delete the gettokenExample and use it in L39

**Comment by daixiang0** [comment]
ping @fisherxu

**Pull request merged**